2/contrib/s2i/run

```diff
@@ -41,6 +41,22 @@ function create_jenkins_credentials_xml() {
   fi
 }
 
+function create_jenkins_config_from_templates() {
+    find ${image_config_dir} -type f -name "*.tpl" -print0 | while IFS= read -r -d '' template_path; do
+        local target_path=${template_path%.tpl}
+        if [[ ! -f "${target_path}" ]]; then
+            if [[ "${target_path}" == "${image_config_path}" ]]; then
+                create_jenkins_config_xml
+            elif [[ "${target_path}" == "${image_config_dir}/credentials.xml" ]]; then
```

Comment:
this should be looking for credentials.xml.tpl right?

Reply:
no, target_path already should have ".tpl" stripped

Reply:
ah, right.